Dockerfile

```diff
@@ -39,11 +39,7 @@ RUN     npm install --production || npm install --production
 COPY     . /calypso
 
 # Build javascript bundles for each environment and change ownership
-RUN     CALYPSO_ENV=wpcalypso make build-wpcalypso && \
-          CALYPSO_ENV=horizon make build-horizon && \
-          CALYPSO_ENV=stage make build-stage && \
-          CALYPSO_ENV=production make build-production && \
-          chown -R nobody /calypso
+RUN	CALYPSO_ENV=production make build && chown -R nobody /calypso
```

Comment:
This need to be further investigated, because we have 3 ways of detecting env in client and server code:
* ```config( 'env )```
* ```NODE_ENV```
* ```CALYPSO_ENV```